classes/cluster/mk22_lab_advanced/openstack/control.yml

```diff
@@ -31,6 +35,18 @@ parameters:
       package:
         python-msgpack:
           version: latest
+      repo:
+      # This repository is needed because the python-influxdb package
+      # required for Mitaka Ceilometer is only present in
+      # mos9.0-proposed from the 9.0-latest repository
+        mirantis_latest_openstack_proposed:
```

Comment:
since this is common with mk22_lab_basic, it might be better to isolate it in a system class (system.linux.system.repo.mos9_latest for instance).